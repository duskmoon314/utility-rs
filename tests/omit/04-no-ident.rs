use utility_types::Omit;

#[derive(Omit, Debug, Clone, PartialEq)]
#[omit(arg(fields(title, author), derive(Debug, PartialEq)))]
struct Article {
    title: String,
    author: String,
    content: String,
    tags: Vec<String>,
}

fn main() {}
