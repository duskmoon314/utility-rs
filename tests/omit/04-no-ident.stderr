error: Missing field `ident`
 --> tests/omit/04-no-ident.rs:4:8
  |
4 | #[omit(arg(fields(title, author), derive(Debug, PartialEq)))]
  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
